dev-docs/best-practices/changelog.md

```diff
@@ -22,19 +22,35 @@ Each file is a markdown file with the following format:
 ```
 # (required) the semver level of this change, one of 'patch', 'minor', or 'major'
 level: ..
+# (required) the audience who this is applicable to, one of 'general', 'admins', 'deployers', 'users', or 'developers' (definied below)
+audience: ..
 # (optional) a reference to the bug or issue tracking this change ("bug xxxxxxx" or "issue xxx")
 reference:  ...
 ---
 <Snippet Text>
 ```
 
+## Audiences
+
+Taskcluster has several groups of people who interact with it. They are defined in [the docs on docs](/dev-docs/best-practices/docs.md).
```

Comment:
docs on docs

Reply:
haha, I wasn't sure what else to call the best practices docs about writing documentation. maybe I should call it "best practices docs about writing documentation" that seems like a better choice